precompiles/distribution/tx.go

```diff
@@ -54,7 +54,7 @@ func (p Precompile) ClaimRewards(
 	}
 
 	validators := p.stakingKeeper.GetDelegatorValidators(ctx, delegatorAddr.Bytes(), maxRetrieve)
-	totalCoins := sdk.Coins{}
+	totalCoins := sdk.NewCoins(sdk.NewCoin(p.stakingKeeper.BondDenom(ctx), sdk.ZeroInt()))
```

Comment:
`sdk.NewCoin` does this panic if `coin.Amount` is zero? I'd refrain from using it because it calls `ValidateDenom` internally